pkg/hintrunner/zero/zerohint_signature.go

```diff
@@ -31,8 +31,6 @@ func newVerifyZeroHint(val, q hinter.ResOperander) hinter.Hinter {
 			//> assert r == 0, f"verify_zero: Invalid input {ids.val.d0, ids.val.d1, ids.val.d2}."
 			//> ids.q = q % PRIME
 
-			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
```

Comment:
why do you remove this comment? its needed, this hint added the python hint line by line in the impl, you should leave it , we import `secPBig`

Reply:
![Screenshot from 2024-06-23 01-46-39](https://github.com/NethermindEth/cairo-vm-go/assets/114918019/b5f105c2-198e-4c09-aa5c-b0cfba124b61)

I suppose this is the hint above. I dont see that extra import anywhere.

Reply:
look line 49 : 

//> q, r = divmod(pack(ids.val, PRIME), SECP_P)

the person who implemented basically wrote 2 times each line of python hint : 
- one time at the beginning of the new function, the whole hint
- another time in the impl line by line 

here, you should leave : 

//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack


because it describes : 

	secPBig, ok := secp_utils.GetSecPBig()
			if !ok {
				return fmt.Errorf("GetSecPBig failed")
			}

Reply:
Well then would it not better to move it just above the following line : 

```
secPBig, ok := secp_utils.GetSecPBig()
		if !ok {
			return fmt.Errorf("GetSecPBig failed")
		}
```

Where it makes sense. Because the current impl seems to suggest that the hint has that extra line of code.

Reply:
you're right lets do that i guess!

Reply:
done.